<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExperimentPoint</class>
 <widget class="QWidget" name="ExperimentPoint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QLineEdit" name="target_edit"/>
   </item>
   <item>
    <widget class="QLineEdit" name="count_edit"/>
   </item>
   <item>
    <widget class="QLineEdit" name="spread_edit"/>
   </item>
   <item>
    <widget class="QPushButton" name="delete_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../Resources/files.qrc">
       <normaloff>:/icons/close.svg</normaloff>:/icons/close.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources/files.qrc"/>
 </resources>
 <connections/>
</ui>
